### PR TITLE
Docs updates!

### PR DESCRIPTION
**Pull request by anurse**
First, I got Codespaces working. Yay! The `bootstrap` skill clones the other repos into the Codespace and gets everything set up for you.

Then, I've updated all the JS, .NET and Python docs.

Finally I reorganized a few of the guides focused on Writing Skills and wrote a new one for Sending Messages.

Take a look! ~~I'll see if I can put a preview together somewhere durable, but I have one live [from my Codespace](https://anurse-aseriousbiz-abbot-docs-pgw4qf7vwr-8080.githubpreview.dev/guides/writing-skills/sending-messages/?tabs=tabid-cs) which will die whenever the Codespace shuts down, or I restart the server :).~~ (this preview is no longer running)

### CODE REVIEW

File: src/guides/writing-skills/sending-messages/index.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+## [C#](#tab/tabid-cs)
+
+```csharp
+await Bot.ReplyAsync("Thanks!");
+```
+
+## [JavaScript](#tab/tabid-js)
+
+```js
+bot.reply("Thanks!");
```
Comment (haacked) [inline src/guides/writing-skills/sending-messages/index.md:19]: Best to check with @pmn and @shana on this but I think it's best to await the reply. I seem to remember getting out of order replies when I didn't. But I never created a clear repro case.

```suggestion
await bot.reply("Thanks!");
```

**Review by haacked — APPROVED** [review]
Fantastic work! Just need to get the build green.

File: src/guides/writing-skills/sending-messages/index.md (hunk 1 above)
Comment (shana) [reply src/guides/writing-skills/sending-messages/index.md:19]: Yes yes, these should be awaited. It doesn't matter if they're not awaited when they're the last instruction to be executed, but as a best practices/examples doc, definitely encouraging awaits is a good thing imo

Comment (anurse) [reply src/guides/writing-skills/sending-messages/index.md:19]: Yep, absolutely agree.

**Pull request merged**